src/handlers/motions/motionCreated/helpers.ts

```diff
@@ -111,7 +116,7 @@ export const writeMintTokensMotionToDB = async (
   const { name, args: actionArgs } = parsedAction;
   const amount = actionArgs[0].toString();
   const tokenAddress = await getColonyTokenAddress(colonyAddress);
-  const motionData = await getMotionData(colonyAddress, motionId, domainId);
+  const motionData = await getMotionData(colonyAddress, transactionHash, logIndex, motionId, domainId, creator);
```

Comment:
Consider refactoring this as well to take an object as an argument. It would make extending it easier since the argument ordering would be irrelevant.